src/move_basic.cpp

```diff
@@ -259,14 +264,14 @@ MoveBasic::MoveBasic(): tfBuffer(ros::Duration(30.0)),
                       reverseWithoutTurningThreshold, 0.5);
 
     nh.param<std::string>("preferred_planning_frame",
-                          preferredPlanningFrame, "map");
+                          preferredPlanningFrame, "goal");
     nh.param<std::string>("alternate_planning_frame",
                           alternatePlanningFrame, "odom");
-    nh.param<std::string>("preferred_planning_frame",
+    nh.param<std::string>("preferred_driving_frame",
                           preferredDrivingFrame, "map");
     nh.param<std::string>("alternate_planning_frame",
                           alternateDrivingFrame, "odom");
-    nh.param<std::string>("base_frame", baseFrame, "base_link");
+    nh.param<std::string>("base_frame", baseFrame, "base_footprint");
 
      nh.param<bool>("verbose_info", verboseInfo, false);
```

Comment:
I didn't realize this was in here, we should really be using ROS_DEBUG and the logger level stuff just to stick to existing standards.

Not something for this PR, but something that should be fixed in the future.